### PR TITLE
bun-canary: Add version 1.0.18-canary.33+61f92d9e5

### PR DESCRIPTION
**Pull request by sitiom**
JavaScript runtime, similar to Node and Deno, designed as a drop-in replacement for Node

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by paperdave** [comment]
hey, i've been working on Bun for windows.

i see you've hardcoded a hash and version number for the canary build. note that this build number changes on every commit, aka multiple times per day. this build will become invalid very soon.

not aware of how scoop works, but is it possible to have an unversioned package? if not, there should not be a package on scoop right now as there is not a stable release with a permalink download.

File: bucket/bun-canary.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/oven-sh/bun/releases/download/canary/bun-windows-x64.zip"
+            }
+        },
+        "hash": {
+            "url": "$baseurl/SHASUMS256.txt"
+        }
+    }
```
Comment (sitiom) [inline bucket/bun-canary.json:40]: > not aware of how scoop works, but is it possible to have an unversioned package? if not, there should not be a package on scoop right now as there is not a stable release with a permalink download.

Hi, this bucket has [a workflow](https://github.com/ScoopInstaller/Versions/blob/master/.github/workflows/excavator.yml) that auto-updates every manifest in this repo every 4 hours, based on the `checkver` and `autoupdate` fields.

In this case, it downloads the latest canary build and fetches the latest revision via `bun --revision`, and fetches the updated hashes in the `SHASUMS256.txt` file if a newer version is found.

Comment (sitiom) [inline bucket/bun-canary.json:40]: > not aware of how scoop works, but is it possible to have an unversioned package? if not, there should not be a package on scoop right now as there is not a stable release with a permalink download.

@paperdave Hi, this bucket has [a workflow](https://github.com/ScoopInstaller/Versions/blob/master/.github/workflows/excavator.yml) that auto-updates every manifest in this repo every 4 hours, based on the `checkver` and `autoupdate` fields.

In this case, it downloads the latest canary build and fetches the latest revision via `bun --revision`, and fetches the updated hashes in the `SHASUMS256.txt` file if a newer version is found.

**Comment by sitiom** [comment]
@paperdave Hash mismatch due to [`SHASUMS256.txt`](https://github.com/oven-sh/bun/releases/download/canary/SHASUMS256.txt) not being updated in the canary builds, can you please fix this?

**Comment by sitiom** [comment]
Removed hash extraction from [`SHASUMS256.txt`](https://github.com/oven-sh/bun/releases/download/canary/SHASUMS256.txt) since it is not being immediately updated.

**Comment by sitiom** [comment]
@HUMORCE maybe you can review this one as well? 🙂

**Comment by HUMORCE** [comment]
checker still works? I cannot tests locally for now. And I noticed that bun has not been added to main bucket yet. It appears to have released version 1.0 a few months ago?

**Comment by sitiom** [comment]
> checker still works? I cannot tests locally for now.

@HUMORCE Updated and fixed.

> And I noticed that bun has not been added to main bucket yet. It appears to have released version 1.0 a few months ago?

Yes, but Windows builds are still in canary. See https://bun.sh/docs/installation#windows

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**